rundeckapp/grails-app/services/rundeck/services/UserService.groovy

```diff
@@ -41,6 +41,7 @@ class UserService {
     public static final String SESSION_ABANDONDED_MINUTES = 'userService.login.track.sessionAbandoned'
     public static final String SHOW_LOGIN_STATUS = 'gui.userSummaryShowLoginStatus'
     public static final String SHOW_LOGGED_USERS_DEFAULT = 'gui.userSummaryShowLoggedUsersDefault'
+    public static final String LDAP_SYNC_USERS_ENABLE = 'security.syncLdapUser'
```

Comment:
i can see this string being used in other places, could this be unified?
EG: RundeckJaasAuthenticationSuccessEventListener

Reply:
This constant needs to be removed